components/AuthButton.tsx

```diff
@@ -42,7 +42,7 @@ export default function AuthButton({ user, height, width }: AuthButtonProps) {
   ) : (
     <Button
       variant="outline"
-      className="inline-flex h-11 animate-shimmer items-center justify-center rounded-md border-2 border-slate-800 bg-[linear-gradient(110deg,#000103,45%,#1e2631,55%,#000103)] bg-[length:200%_100%] px-12 font-medium  text-white text-lg transition-colors focus:outline-none focus:ring-2 focus:ring-slate-400 focus:ring-offset-2 focus:ring-offset-slate-50"
+      className="inline-flex h-11 animate-shimmer items-center justify-center rounded-md border-2 border-slate-800 bg-[linear-gradient(110deg,#000103,45%,#1e2631,55%,#000103)] hover:text-white  bg-[length:200%_100%] px-12 font-medium  text-white text-lg transition-colors focus:outline-none focus:ring-2 focus:ring-slate-400 focus:ring-offset-2 focus:ring-offset-slate-50 "
```

Comment:
remove the changes here. Not part of the PR

Reply:
Yes, I did it